src/components/context-menu/ContextMenu.svelte

```diff
@@ -1,7 +1,19 @@
 <svelte:options accessors={true} immutable={true} />
 
+<script lang="ts" context="module">
+  type HideFns = Set<() => void>;
+  const hideFns: HideFns = new Set<() => void>();
+
+  export function hideAllMenus() {
+    // TODO: https://github.com/sveltejs/language-tools/issues/1229
```

Comment:
Know this came from Menu but not sure I understand what the issue has to do with this bit of code?

Reply:
Good point. I'm not sure this comment is still relevant for both components